ckanext/unhcr/controllers/extended_group.py

```diff
@@ -0,0 +1,15 @@
+import logging
+from ckan import model
+import ckan.plugins.toolkit as toolkit
+from ckan.controllers.group import GroupController
+log = logging.getLogger(__name__)
+
+
+class ExtendedGroupController(GroupController):
+
+    # Read
+
+    def read(self, id):
+        if not toolkit.c.user:
+            return toolkit.render('page.html')
```

Comment:
@roll, to show the please login page better to raise a 403 as you suggest:

```suggestion
            return toolkit.abort(403, 'Message here!')
```